### ISSUE TITLE
Mock-up main screen for desktop.

### ISSUE THREAD

**Issue by desmondmullen** [charlesmbrady/Project3#22]

**Comment by charlesmbrady** [comment]
Kinda think we should just design for mobile.  At least that's what I'm going to do first.  Don't see how it would be super useful MVPwise as a desktop app.

**Comment by desmondmullen** [comment]
History is the kind of thing you’d look at on desktop and if we show that we used responsive design principles and make it look good on both we’ll definitely get some additional “points”.

Good design uses relative numbers (%, em, rem, sometimes vh and vw) and then adding some media queries that rearrange elements is easy. I’ve tried to make most of my homeworks responsive so they look good on both desktop and mobile.

Check out https://clickypiccy.com on both desktop and mobile for an example. Then look at the code, the only differences are media queries in style.css.

I did responsive design on the main page of goTreeUrself but Tim was the main front-end guy and we didn’t really port it to mobile. Still, the front page works: https://gotreeurself.herokuapp.com/

-DM
-- 
Desmond Mullen 919/260-8858
dsm@busylittlestudios.com
